### PR TITLE
Add constant line function

### PR DESCRIPTION
**Pull request by agao48**
**Describe your changes**
Adds the constantLine function

**Testing performed**
Test files added with [mtcmptest](https://github.com/shanson7/mtcmptest) used locally

### CODE REVIEW

**Comment by agao48** [comment]
Previously created https://github.com/grafana/metrictank/pull/1730, but new commits weren't showing up due to some git branch wonkiness on our end.

File: expr/func_constantline.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			ArgFloat{key: "value", val: &s.value},
+		}, []Arg{
+			ArgSeriesList{},
+		}
+}
+
+func (s *FuncConstantLine) Context(context Context) Context {
+	s.from = context.from - 1
+	s.to = context.to - 1
+	return context
```
Comment (Dieterbe) [inline expr/func_constantline.go]: within MT, `from` is always inclusive, so the first point can have ts from, but nothing earlier, as that would be out of range.
also the names from/to are by convention used to mark the time window.
if you want to track the first and last timestamp to include, better names would be first and last

Comment (agao48) [reply expr/func_constantline.go]: Code was hitting https://github.com/grafana/metrictank/blob/master/api/graphite.go#L1363-L1367, and I was trying to line up with graphite's constantLine behavior, which was inclusive on both ends (even though normally, graphite is from exclusive, to inclusive).

Comment (Dieterbe) [reply expr/func_constantline.go:31]: I see. in that case i propose we just add a comment explaining that graphite does it non-standard and we do the same.

**Comment by Dieterbe** [comment]
thanks!

**Pull request merged**